istio-telemetry/kiali/values.yaml

```diff
@@ -65,18 +67,26 @@ kiali:
       #   ldap_user_id_key: "cn"
 
     secretName: kiali # You must create a secret with this name - one is not provided out-of-box unless you've chose to create a demo secret.
-    usernameKey: username # This is the key name within the secret whose value is the actual username.
-    passphraseKey: passphrase # This is the key name within the secret whose value is the actual passphrase.
```

Comment:
are these key/value pairs deprecated?

Reply:
Yes, they have not been used/needed for a long time now (at least since Kiali 0.16). These were obsolete since March 2019 due to this PR merge: https://istio.io/docs/tasks/observability/kiali/#create-a-secret

So these are remnants and should have been removed a while ago.